CHANGELOG.md

```diff
@@ -1 +1,107 @@
 # Changelog
+
+Items starting with `DEPRECATE` are important deprecation notices. For more
+information on the list of deprecated flags and APIs please have a look at
+https://docs.docker.com/engine/deprecated/ where target removal dates can also
+be found.
+
+## 18.02.0-ce (2018-02-DD)
+
+### ---Triage---
```

Comment:
What is this heading?

Reply:
Victor Vieux created a [tool](https://github.com/vieux/docker-changelog) that helps create the changelog. 

The `triage` label is added because the tool itself doesn't categorize every change.  I'm still working through categorizing most of the objects in triage, so what will be left is hopefully a small list of changes that I'm unsure about with regards to where they fit in.

To be clear, I am still in the process of preparing the changelog before it is ready for review. Sorry for the confusion. I'll be sure to let you know when I am done.